pages/index.html

```diff
@@ -94,7 +94,7 @@ <h2><a class="headline-link" href="#opensource">{% trans "Free &amp; Open Source
       {% include "donate_paypal.html" %}
     </div>
     <div>
-      <a class="button button-secondary" href="{% url "/features.html" %}">{% trans "More Features" %}</a>
+      <a class="button button-secondary" href="{% url "/get-involved.html" %}">{% trans "Get Involved" %}</a>
```

Comment:
After putting get involved, the get involved link will be the reduntant one, we just have to prioritise which one to show in this section